app/components/nav-bar.js

```diff
@@ -9,6 +9,12 @@ export default class NavBar extends Component {
     return !(String(this.session.currentRouteName).includes('public'));
   }
 
+  @computed('session.currentRouteName')
+  get isNotWizardPageRoute() {
+    return !(String(this.session.currentRouteName).includes('edit'))
+    && !(String(this.session.currentRouteName) === 'create');
```

Comment:
```suggestion
    && String(this.session.currentRouteName) !== 'create';
```

Reply:
Both implementation are the same?? Then why this?? Coding standard/covention??

Reply:
Yes